Application/EdFi.Ods.Features/OpenApiMetadata/Strategies/FactoryStrategies/OpenApiMetadataCompositePathsFactoryStrategy.cs

```diff
@@ -17,107 +17,111 @@
+    public class OpenApiMetadataCompositePathsFactoryStrategy : IOpenApiMetadataPathsFactorySelectorStrategy
     {
         private readonly XElement _defaultCollectionRoute = XElement.Parse("<Route relativeRouteTemplate='/{compositeName}' />");
         private readonly XElement _defaultItemRoute = XElement.Parse("<Route relativeRouteTemplate='/{compositeName}/{id}' />");
-        private readonly SwaggerDocumentContext _swaggerDocumentContext;
+        private readonly OpenApiMetadataDocumentContext _openApiMetadataDocumentContext;
 
-        public SwaggerCompositePathsFactoryStrategy(SwaggerDocumentContext swaggerDocumentContext)
+        public OpenApiMetadataCompositePathsFactoryStrategy(OpenApiMetadataDocumentContext openApiMetadataDocumentContext)
         {
-            _swaggerDocumentContext = swaggerDocumentContext;
+            _openApiMetadataDocumentContext = openApiMetadataDocumentContext;
         }
 
-        public IEnumerable<SwaggerPathsResource> ApplyStrategy(IEnumerable<SwaggerResource> swaggerResources)
+        public IEnumerable<OpenApiMetadataPathsResource> ApplyStrategy(IEnumerable<OpenApiMetadataResource> openApiMetadataResources)
         {
-            var swaggerResourceList = swaggerResources.Select(
-                                                           r => new SwaggerPathsResource(r.Resource)
-                                                                {
-                                                                    CompositeResourceContext =
+            var openApiMetadataResourceList = openApiMetadataResources.Select(
+                                                           r => new OpenApiMetadataPathsResource(r.Resource)
+                                                           {
+                                                               CompositeResourceContext =
                                                                         r.CompositeResourceContext,
-                                                                    Readable = true, Path =
-                                                                        $"/{_swaggerDocumentContext.CompositeContext.CategoryName}/{r.Resource.PluralName}",
-                                                                    RequestProperties = GetMetaDataPropertiesForCurrentResource(r)
-                                                                })
+                                                               Readable = true,
+                                                               Path =
+                                                                        $"/{_openApiMetadataDocumentContext.CompositeContext.CategoryName}/{r.Resource.PluralName}",
+                                                               RequestProperties = GetMetaDataPropertiesForCurrentResource(r)
+                                                           })
```

Comment:
this whole section needs to be reformatted correctly, also there is extra blank lines in it.

Reply:
Fixed